app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1825,3 +1825,101 @@ table.orders{
   font-size: 24px;
   margin-top: 10px;
 }
+
+h1.admin-welcome {
+  border: 1px solid red;
+  text-align: center;
+  margin: 50px 0px 10px 0px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `50px 0px 10px` instead of `50px 0px 10px 0px`<br>`0px` should be written without units as `0`